Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocControlObject.cs

```diff
@@ -32,6 +32,9 @@ namespace Remotion.ObjectBinding.Web.Development.WebTesting.ControlObjects
   /// </summary>
   public abstract class BocControlObject : WebFormsControlObjectWithDiagnosticMetadata
   {
+    private const string c_couldNotFindValidationErrorFieldExceptionMessage =
+        "Could not find validation error field. This could be due to wrong markup or a missing validators.";

```

Comment:
Little grammar error
```suggestion
        "Could not find validation error field. This could be due to wrong markup or missing validators.";
```

Reply:
Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/11/commits/dafe24eb39dd3d5dd0a97ae42a728cd48f7e7479.